_generated/CopyFilesV2.versionmap.txt

```diff
@@ -1,2 +1 @@
-Default|2.231.0
-Node20-228|2.231.2
+Default|2.231.2
```

Comment:
Since the build configs are deleted, versionmap.txt should be deleted too

Reply:
updated